How to configure external database during server setup (#2004)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/server-setup.adoc

When the channel synchronization process is complete, you can continue with client registration.
For more instructions, see xref:client-configuration:registration-overview.adoc[].

== Set up {productname} with external database

In this example, we use RDS product from Amazon Web Service.

[WARNING]
====
Currently, configuring an external database is not supported by yast2 setup.

You can configure an external database using the [command]``mgr-setup`` command line tool.

====
This section guides you through {productname} setup using [command]``mgr-setup``.

.Procedure : Configuring server with external database

> First create a [path]``setup_env.sh`` file in the [path]``/root`` directory defining the server setup like this:

Reviewer: ```suggestion
. First create a [path]``setup_env.sh`` file in the [path]``/root`` directory defining the server setup.
  The details are explained in the following steps.
```

> . Here you define your certificate information and password.

Reviewer: ```suggestion
. Set the variables defining your certificate information and password.
```

> The information are the same as <<proc-quickstart-certificate-information-yast>>.

Reviewer: ```suggestion
The information is the same as explained at <<proc-quickstart-certificate-information-yast>>.
```

+
----
CERT_O="SUSE"
CERT_OU="SUSE"
CERT_CITY="N"
CERT_STATE="B"
CERT_COUNTRY="DE"
CERT_EMAIL="email@prov.com"
CERT_PASS="spacewalk"
USE_EXISTING_CERTS="N"
----

. Define your database.
The user and password will be created during the setup.
If you are using an external database, specify the hostname and port.
+
----
MANAGER_USER="spacewalk"
MANAGER_PASS="spacewalk"
MANAGER_ADMIN_EMAIL="email@prov.com"
MANAGER_DB_NAME="susemanager"
MANAGER_DB_HOST="db hostname"
MANAGER_DB_PORT="db port"
MANAGER_DB_PROTOCOL="TCP"
MANAGER_ENABLE_TFTP="Y"
----

> . Define the specific configuration for the external database.

Reviewer: ```suggestion
. Set the variables defining the specific configuration for the external database.
```
